qualtran/linalg/testing.py

```diff
@@ -0,0 +1,20 @@
+#  Copyright 2024 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import numpy as np
+from numpy.typing import NDArray
+
+
+def assert_matrices_almost_equal(A: NDArray, B: NDArray, *, atol: float = 1e-5):
```

Comment:
docstring? worth mentioning that this is based on the norm which is unlike e.g. `np.testing.assert_almost_equal`

Reply:
added docstring